README.md

```diff
@@ -20,15 +20,27 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 ### Screenshots
 
-![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot1.png "Main page")
+![Screenshot of main page](nautobot/docs/media/screenshot1.png "Main page")
```

Comment:
IIRC these need to be absolute links for the README to render correctly on PyPI (i.e. https://pypi.org/project/nautobot/)

Reply:
Ah ok, thanks I'll revert them.